highlight.io/components/QuickstartContent/logging/python/otel.tsx

```diff
@@ -55,22 +61,65 @@ console_handler = logging.StreamHandler(sys.stdout)
 console_handler.setLevel(logging.DEBUG)
 formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
 console_handler.setFormatter(formatter)
-logger.addHandler(console_handler)`,
+logger.addHandler(console_handler)
+					`,
 					language: 'python',
 				},
 			],
 		},
 		{
-			title: 'Send logs using OpenTelemetry!',
+			title: 'Set up OpenTelemetry for tracing.',
```

Comment:
technically under the `Backend: Logging` section so this is a weird here, but will be refactored soon with @ysadamt 's work so should be fine for now